pvlib/pvsystem.py

```diff
@@ -20,6 +21,11 @@
 from pvlib.tools import _build_kwargs
 from pvlib.location import Location
 from pvlib import irradiance, atmosphere
+from pvlib import singlediode_methods
+
+# expose single diode methods to API
+bishop88 = singlediode_methods.bishop88
+estimate_voc = singlediode_methods.estimate_voc
```

Comment:
I don't know? I guess 5 months ago I thought that these two (`bishop88`, `estimate_voc`) were the only new methods worth exposing, and since everything else is in `pvlib.pvsystem` it made sense to add them to the API there.

I guess 5 months later that doesn't make sense anymore, especially if we're planning on moving more low-level functions like `lambertw` over to `pvlib.singlediode_methods`.

If this is what everyone wants, then I will make it happen.

Reply:
this particular comment about the import syntax rather than the api itself. is there a difference between ``from a import b`` and ``import a; b = a.b``?

Reply:
✔️

Reply:
AFAICT no difference, but if your IDE is linting and you don't use an import, eg: `estimate_voc` then it warns you, but if you define it as a variable then it doesn't.